doc/source/ray-observability/ray-logging.rst

```diff
@@ -77,10 +111,17 @@ This produces the following output:
     (MyActor(index=2) pid=482120) hello there
     (MyActor(index=1) pid=482119) hello there
 
-How to set up loggers
+Customizing worker loggers
 ~~~~~~~~~~~~~~~~~~~~~
-When using ray, all of the tasks and actors are executed remotely in Ray's worker processes. 
-Since Python logger module creates a singleton logger per process, loggers should be configured on per task/actor basis. 
+When using ray, all of the tasks and actors are executed remotely in Ray's worker processes.
+
+If a user intends to log a message via a worker, a logger should be used. Typically the way this is done is by getting a logger using the filename where the log is generated; see `the Python logging documentation <https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial>`__ for more information about this.
+
+Workers will by default use the standard Ray logging configuration given in `python/ray/log.py` for code that is internal to Ray. Since the Python logger module creates a singleton logger per process, if a worker needs to log a message they should configure logging on a per task/actor basis.
```

Comment:
```suggestion
By default, workers use the standard Ray logging configuration given in `python/ray/log.py` for code that is internal to Ray. Because the Python logger module creates a singleton logger per process, if a worker needs to log a message they should configure logging on a per task or actor basis.
```